docs/zookeeperAdmin.html

```diff
@@ -1446,7 +1446,17 @@ <h3 class="h4">Configuration Parameters</h3>
               </pre>
 </dd>
 
-        
+<dt>
+<term>tcpKeepAlive</term>
+</dt>
+<dd>
+<p>(Java system property: <strong>zookeeper.tcpKeepAlive</strong>)</p>
```

Comment:
Without digging into TCP options too deeply, I did the best I could.  Thanks for the suggestion.